functions/go/set-gcp-resource-ids/main.go

```diff
@@ -10,55 +10,32 @@ import (
 	"k8s.io/apimachinery/pkg/runtime/schema"
 )
 
+var (
+	Folder = schema.GroupVersionKind{Group: "resourcemanager.cnrm.cloud.google.com", Version: "v1beta1", Kind: "Folder"}
+	GCPProject = schema.GroupVersionKind{Group: "resourcemanager.cnrm.cloud.google.com", Version: "v1beta1", Kind: "Project"}
+	ConfigControllerContext = schema.GroupVersionKind{Group: "core.cnrm.cloud.google.com", Version: "v1beta1", Kind: "ConfigConnectorContext"}
+	)
+
 func main() {
 	// TODO: fn.AsMain should support an "easy mode" where it runs against a directory
-	if err := fn.AsMain(fn.ResourceListProcessorFunc(Run)); err != nil {
```

Comment:
Some future work: pass the context to `SetGCPProject` and leverage the context to log results.